tests/lib/tools/to-one-line

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+show_help() {
+    echo "usage: to-one-line [string]"
+}
+
+to_one_line(){
+    echo "$1" | tr '\r\n' ' ' | tr -s ' '
+}
+
+if [ $# -eq 0 ]; then
+    show_help
+    exit 1
+fi
+
+if [ $# -eq 1 ] && [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
```

Comment:
Could you please make it so that it prints help when invoked without the arguments *and* to organize the conditions in a way that doesn't make operator priority a problem. Note that this has incorrect semantics that just happens to do what we need as the parentheses are `( a && b ) || c`, which is not what we want.

```suggestion
if [ $# -eq 0 ] || [ "${1:-}" = "-h" ] || [ "${1:-}" = "--help" ]; then
```

Reply:
done